ui-v2/src/components/ui/data-table.tsx

```diff
@@ -6,29 +13,24 @@ import {
 	TableHeader,
 	TableRow,
 } from "@/components/ui/table";
-import {
-	Select,
-	SelectContent,
-	SelectItem,
-	SelectTrigger,
-	SelectValue,
-} from "@/components/ui/select";
 import { type Table as TanstackTable, flexRender } from "@tanstack/react-table";
 
 import {
 	Pagination,
 	PaginationContent,
+	PaginationFirstButton,
 	PaginationItem,
+	PaginationLastButton,
 	PaginationNextButton,
 	PaginationPreviousButton,
-	PaginationFirstButton,
-	PaginationLastButton,
 } from "@/components/ui/pagination";
 import { cn } from "@/lib/utils";
 
 export function DataTable<TData>({
+	hidePagination = false,
```

Comment:
Even though the current concurrency limit tables don't have pagination, I think we should include pagination with all tables in the v2 UI to make sure we aren't over fetching data.

cc @aaazzam in case you have any opinions since adding pagination to the concurrency tables would be a diversion from the current UI.

Reply:
Let me revert the hidePagination prop, and I'll tackle pagination last in another PR.

Updated latest in latest push

Reply:
ohh looks like there's no `/count` support for this endpoint.
Is that needed for the pagination feature?